HashRegistrarSimplified.sol

```diff
@@ -12,17 +12,20 @@ and all ether still locked after 8 years will become unreachable.
 
 The plan is to test the basic features and then move to a new contract in at most
 2 years, when some sort of renewal mechanism will be enabled.
-
 */
 
+
 import 'ENS.sol';
 
+/*
+The Deed is a contract intended simply to hold ether
```

Comment:
Nit: This should have *s on the left of each line - and should probably be a part of the following comment block.